### PR TITLE
dooit: Fix textual version

### PR DESCRIPTION
**Pull request by kraanzu**
## Description of changes

Fix: Dooit's dep textual

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by khaneliman** [comment]
Can you reword the commit, please. `dooit: fix textual version`

**Comment by khaneliman** [comment]
Running nixpkgs-review now, since it's your application I'll trust the version change fixes the behavior to your expectation.

**Comment by kraanzu** [comment]
Hmmm... I did the changes you mentioned and its still not building. Something with version, I'll check again. Sorry for the trouble :(

File: pkgs/by-name/do/dooit/package.nix [hunk 1]
```diff
@@ -34,6 +33,21 @@ python3.pkgs.buildPythonApplication rec {
     python-dateutil
     pyyaml
     textual
```
Comment (khaneliman) [inline pkgs/by-name/do/dooit/package.nix:35]:
```suggestion
```

Comment (khaneliman) [reply pkgs/by-name/do/dooit/package.nix:35]: Looks like the other reference got left in there, so it's picking up the new version, as well.

Comment (kraanzu) [reply pkgs/by-name/do/dooit/package.nix:35]: oh lol hahaha

**Comment by khaneliman** [comment]
Alright, kicking off a review. Hopefully everything works, thanks for taking this on!

**Comment by kraanzu** [comment]
Alright. Its fine now. Thanks a lot for your time! much appreciated

**Comment by khaneliman** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 345228`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>dooit</li>
    <li>dooit.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>dooit</li>
    <li>dooit.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>dooit</li>
    <li>dooit.dist</li>
  </ul>
</details>

**Review by khaneliman — APPROVED** [review]

**Review by wesleyjrz — APPROVED** [review]
[`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review) result on `x86_64-linux`.

<details>
<summary>2 packages built:</summary>

* dooit 
* dooit.dist

</details>

**Comment by khaneliman** [comment]
Looks like dooit is broken on master now from a textual update. This fixes it not even running, too.

**Pull request merged**